allennlp/tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -258,7 +269,7 @@ def test_token_idx_sentence_pairs(self):
             ".",
             "</s>",
             "</s>",
-            "It",
+            "ĠIt",
```

Comment:
Do you know why it's adding spaces to the first token?  Is that expected?

Reply:
I think it's a change in defaults.

At first, people thought that `Ġ` means that this is the beginning of the word (opposite of `##` in BERT). So then people were complaining about the "bug" where the `Ġ` wasn't there at the beginning of the first token, and they made a fix. But now it means "space", so not having it makes sense ... I want to not get involved and just stick with the huggingface default.